develop/app/Database/Seeds/Wallet.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Database\Seeds;
+
+use CodeIgniter\Database\Seeder;
+use Faker\Factory;
+
+class Wallet extends Seeder
+{
+    public function run()
+    {
+        $faker = Factory::create();
+        $now   = date("Y-m-d H:i:s");
+
+        for ($i = 1; $i < 5; $i++) {
+            $data = [
+                'u_key'      => $i,
+                'balance'    => $faker->numberBetween(500, 100000),
```

Comment:
I think we need maybe have three users, the first one have 50000000 (The balance will not make the compensation occur), the second is also and the third have 0 balance (It can make sure when we use the third user, the compensate will happen successfully).
This design will make our experiment eaiser and more convenient.

Reply:
This chahges is done.